scaffolding-chef-inspec/tests/user-linux-default/inspec.yml

```diff
@@ -0,0 +1,8 @@
+name: user-linux-default
+title: Effortless Audit with default plan settings
+maintainer: CCP Team
```

Comment:
Please change to: `The Habitat Maintainers humans@habitat.sh`

Reply:
I don't think that is the correct maintainer. This is not a project that is part of the habitat project.

Reply:
I see -- that's the maintainer we have for every other core plan and plan in the repo. What do you think the correct maintainer should be?

Reply:
This is not a core plan this is a custom library to help customers use our tools easier to use and to reduce the need for other projects that do similar things from needing to maintain the same code.